components/automate-ui/src/app/modules/token/token-details/api-token-details.component.html

```diff
@@ -63,7 +63,7 @@
           </chef-form-field>
         </form>
         <div id="save">
-          <chef-button primary [disabled]="saveInProgress || !updateForm.valid || !this.updateForm.dirty" (click)="saveToken()">
+          <chef-button primary [disabled]="saveInProgress || !updateForm.valid || !updateForm.dirty" (click)="saveToken()">
```

Comment:
If this is critical change... well, obviously it is since it is the only change, then it should theoretically be required for these, too
https://github.com/chef/automate/blob/f043f5ad9430c4ba3ad0d32b39b7df2fb09f609e/components/automate-ui/src/app/pages/data-feed-details/data-feed-details.component.html#L49
https://github.com/chef/automate/blob/f043f5ad9430c4ba3ad0d32b39b7df2fb09f609e/components/automate-ui/src/app/pages/data-feed-details/data-feed-details.component.html#L49

Reply:
You are correct!  Good idea to sweep for that 🙇